Makefile

```diff
@@ -148,25 +150,10 @@ manifests: controller-gen ## Generate manifests e.g. CRD, RBAC etc.
 generate-go: $(COUNTERFEITER) ## Generate code via go generate.
 	PATH=$(abspath hack/tools/bin):$(PATH) go generate ./...
 
-generate: controller-gen ## Generate code via controller-gen
+generate: tools ## Generate code via controller-gen
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt",year=$(shell date +%Y) paths="./..."
 	$(MAKE) fmt
 
-controller-gen: ## Download controller-gen
```

Comment:
Should we remove this from the other Makefiles in this repo? Maybe as part of a followup PR?

Reply:
Let's do as a follow up and create an issue. Think it might need a bigger refactoring, and realistically not going to have time this week. This PR at least stopped all the main manifests being regenerated with the wrong controller-gen version when i changed a core package in another PR.

Reply:
Removed the other ones as per zoom call.